.NET/Microsoft.Recognizers.Text.DateTime/Parsers/BaseTimeZoneParser.cs

```diff
@@ -0,0 +1,130 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.RegularExpressions;
+using DateObject = System.DateTime;
+using Microsoft.Recognizers.Definitions.English;
+using Microsoft.Recognizers.Text.Number;
+
+namespace Microsoft.Recognizers.Text.DateTime
+{
+    public class BaseTimeZoneParser : IDateTimeParser
+    {
+        public static readonly string ParserName = Constants.SYS_DATETIME_TIMEZONE; //"TimeZone";
+
+        public BaseTimeZoneParser()
+        {
+        }
+
+        public ParseResult Parse(ExtractResult result)
+        {
+            return this.Parse(result, DateObject.Now);
+        }
+
+        // GetMinutes compute minutes shift from UTC 0 from matched numbers in texts. e.g. "-4:30" -> -270; "8"-> 480; "+8"-> 480
+        public int ComputeMinutes(string utcOffset)
+        {
+            if (utcOffset.Length == 0)
+            {
+                return Constants.InvalidOffsetValue;
+            }
+
+            int sign = 1; // earlier than utc, default value
```

Comment:
Please define two constants for 1 and -1 in this case.
Also, the code comments are incorrect.